util/git/client.go

```diff
@@ -316,18 +319,42 @@ func (m *nativeGitClient) Init() error {
 	if err != nil {
 		return err
 	}
-	repo, err := git.PlainInit(m.root, false)
+
+	_, err = m.runCmd("init")
 	if err != nil {
 		return err
 	}
-	_, err = repo.CreateRemote(&config.RemoteConfig{
-		Name: git.DefaultRemoteName,
-		URLs: []string{m.repoURL},
-	})
-	return err
+
+	_, err = m.runCmd("remote", "add", git.DefaultRemoteName, m.repoURL)
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
I `HAD` to do this to make things work! Whatever that package is doing, it doesn't work with sparse checkout